packages/web-pkg/src/components/FilesList/ResourceLink.vue

```diff
@@ -90,15 +93,17 @@ export default {
     }
   },
   methods: {
-    emitClick() {
+    emitClick(e: unknown) {
+      if (!e || typeof (e as Event).stopPropagation !== 'function') {
```

Comment:
```suggestion
    emitClick(e: MouseEvent) {
      if (!e || typeof e.stopPropagation !== 'function') {
```

Also, why do you need to check for `stopPropagation` here?